services/command-router/src/main/java/org/eclipse/hono/commandrouter/CommandRouterService.java

```diff
@@ -88,7 +89,7 @@ Future<CommandRouterResult> setLastKnownGatewayForDevice(String tenantId, Map<St
      *         The <em>status</em> will be <em>204 No Content</em> if the operation completed successfully.
      * @throws NullPointerException if any of the parameters except lifespan is {@code null}.
      */
-    Future<CommandRouterResult> registerCommandConsumer(String tenantId, String deviceId,
+    Future<CommandRouterResult> registerCommandConsumer(String tenantId, String deviceId, boolean sendEvent,
```

Comment:
The [Command Router API documentation](https://www.eclipse.org/hono/docs/api/command-router/) needs to be adapted concerning the added parameter.

In the [Device Notifications documentation page](https://www.eclipse.org/hono/docs/concepts/device-notifications/), it should also be noted that sending of the (dis)connected ttd events is done via the Command Router for MQTT adapter (and with follow-up PR also AMQP adapter).

Reply:
done, please take a look.